stestr/commands/run.py

```diff
@@ -308,6 +308,11 @@ def run_command(config='.stestr.conf', repo_type='file',
             exit(1)
         repo = util.get_repo_initialise(repo_type, repo_url)
     combine_id = None
+    if concurrency and concurrency < 0:
+        msg = ("The provided concurrency value: %s is not valid. An integer "
+               ">= 0 must be used.\n")
```

Comment:
heh, 
 `">= 0 must be used.\n" % concurrency)` 
?

Reply:
Heh, oops I'll fix that now